### DISCUSSION TITLE
Frigate hardware with coral.

### DISCUSSION THREAD

**Discussion by joelegg** [blakeblackshear/frigate#4204] [Ask A Question]
Hi all. I have a Synology 1019+ dsm7 but can't get frigate working with my coral. (Via vmm or in container ). 

What would the community recommend hardware server wise  to use with my coral for frigate. I'd want to connect to my ha server running on my Synology. 

Thanks very much.

**Comment by joelegg** [comment]
Ended up getting a dell optiplex 7050m 

Now to try find a tutorial to set up with Ubuntu. Hopefully I can easily have it connect to my Synology running home assistant

**Reply by Steve973** [reply]
I had to turn off protected mode in the frigate addon settings to get frigate to find my coral USB.

**Reply by joelegg** [reply]
Thanks. Is this on a Synology?
